### PR TITLE
7166 accessibility index 5 2 rebased 20210628

### PR DESCRIPTION
**Pull request by phudson-he**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change
Changes have been done to ensure [WCAG 2.1 requirements](https://www.w3.org/TR/WCAG21/#requirements-for-wcag-2-1) have been met for accessibility guidelines.

Details of the changes are as follows:

- Removed empty class="" and style="" attributes - in order to remove W3C html validation errors
- For usability reasons, merged the 2 top menus into 1
- Removed inline styling and added relevant css classes
- Ensured correct header ordering
- Added html5 semantic containers - header, main, footer
- Ensured there are no colour contrast errors
- Ensured headers are nested correctly
- On smaller screen widths, made sure that no page elements are overlapping

### Issues Solved
Ensures the index page conforms to accessibility guidelines
#

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: Historic England
*   Found by: @phudson-he
*   Tested by: @phudson-he
*   Designed by: @phudson-he

### Further comments

Previous changes in PR #7166 included, with further changes included to move accessibility specific css into the new accessibility.css file

### CODE REVIEW

**Comment by phudson-he** [comment]
@robgaston As requested, thanks

**Comment by aj-he** [comment]
@robgaston can I ask for this to be reviewed please - @phudson-he has a number of PR to rebase that is dependent on it. Thx

File: arches/install/arches-templates/project_name/templates/index.htm [hunk 1]
```diff
@@ -42,344 +42,313 @@
     <link href="{% static 'css/unify.css' %}" rel="stylesheet">
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.2.0/css/font-awesome.min.css">
     <link href="{% static 'css/index.css' %}" rel="stylesheet">
-
```
Comment (robgaston) [inline arches/install/arches-templates/project_name/templates/index.htm:45]: I notice that the include of the accessibility.css file is missing here.  Could you please add that and ensure that this file is consistent with the main index.html file?

**Review by robgaston — CHANGES_REQUESTED** [review]
I notice that there are some inconsistencies between the main and project template index.html files that need to be addressed.

Additionally, I've noticed that some of the styles don't seem to be working on the page as before, regardless of whether I have ACCESSIBILITY_MODE turned on.  Specifically, the flexing of images on the splash page seems to be broken now, so that images have a width that forces the page to horizontally scroll.

Could you please take a look at the above issues and address them?

File: arches/install/arches-templates/project_name/templates/index.htm (hunk 1 above)
Comment (phudson-he) [reply arches/install/arches-templates/project_name/templates/index.htm:45]: @robgaston Image flexing, fixes for missing styling, added the accessibility.css include back in and ensured the install index.htm was also updated, all sorted and changes committed. Will resolve this conversation once our tester has passed the fixes.

**Comment by phudson-he** [comment]
@robgaston Our tester has passed the tests/checks for this, so hopefully all requirements have now been met

**Review by robgaston — APPROVED** [review]
These changes look good to me.  Thanks, @phudson-he

**Pull request merged**